src/olympia/blocklist/templates/admin/blocklist/includes/enhanced_blocks.html

```diff
@@ -5,10 +5,8 @@ <h3>{% blocktrans with count=blocks|length %}{{ count }} Add-on GUIDs{% endblock
         {% for block_obj in blocks %}
           <li>
             {{ block_obj.guid }}.
-            {% if block_obj.addon %}
-              <span class="addon-name">{{ block_obj.addon.name }}</span>
-              ({{ block_obj.addon.average_daily_users }} {% trans 'users' %}).
-            {% endif %}
+            <span class="addon-name">{{ block_obj.addon.name }}</span>
```

Comment:
Why did we need an `if` statement to check for `block_obj.addon` before, but no longer need it? We're still accessing `block_obj.addon.name`.

Reply:
`block_obj.addon` won't exist now when the `block_obj` is a `FakeBlock` (i.e. when a large number of blocks are loaded).  Template parsing just ends up resolving missing attributes to `''` so it was in an `if` so it didn't look like ` users` when addon didn't exist.